client/reader/search-stream/style.scss

```diff
@@ -350,7 +350,7 @@
 	.reader-share__button-label,
 	.comment-button__label-status,
 	.like-button__label-status {
-		@include screen-reader-text();
+		@include hide-content-accessibly();
 	}
```

Comment:
Should we turn this into a `.hide-content-accessibly` reusable class? It won't work for the breakpoint-specific ones, but it would be useful as a utility class for these instances.

Reply:
That's a good shout. I think the Calypso linter will grumble about it though because it doesn't follow the class naming guidelines. Wonder if we can add an exception somewhere...